.pre-commit-config.yaml

```diff
@@ -35,8 +33,8 @@ repos:
       - id: check-vcs-permalinks
         name: "Check that VCS links are permalinks"
 
-      - id: check-ast
-        name: "Check Python AST"
+      # - id: check-ast
```

Comment:
Should be removed or uncommented

Reply:
I commented it as a stopgap fix for #2402 , but depending on what we ultimately decide there, there's a chance we may want to restore it. That said, I can remove it, if you think its important.